src/Compilers/CSharp/Test/Syntax/Parsing/UsingDirectiveParsingTests.cs

```diff
@@ -5257,4 +5258,143 @@ public void UsingStatic_UnsafeType_CSharp12_UnsafeFlag()
         }
         EOF();
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/67989")]
+    public void AliasToPointerReferencedInExpression()
```

Comment:
I would like to see a test in the semantic layer that compiles successfully.

Reply:
can you give an example of what you mean?

Reply:
> can you give an example of what you mean?  

```nameof(Alias)``` comes to mind, for example.

Reply:
Assuming, it goes through the affected code path.